### PR TITLE
Added OdileSus.html

### PR DESCRIPTION
**Pull request by memeseller**
Put together the dialogue for Odile's event where she figures out (almost all) of the time loop.

### CODE REVIEW

**Review by memeseller — COMMENTED** [review]
Fixed some typos

**Comment by goldstargloww** [comment]
nice :D glancing over it quickly it looks mostly good, but i'll take a more thorough look when i get home

one thing though, it doesn't have the new dialogue formatting, which isn't a _necessity_, but i think it'd be nice to get that done before pushing this one to main (for practice if nothing else, i'm still pretty new to actually collaborating myself on github)

i'm pretty sure others should be able to contribute to this pull request now? so if you don't wanna handle the new formatting, me and/or feli can do it ^^

(also im not in charge here these are just suggestions hshhdhd)

**Comment by memeseller** [comment]
Ahhh apologies, I didn't realize there was new formatting, I think I just clicked a page at random to reference and that must've had the old formatting. I can take care of editing this in a few hours if you point me towards an example of the new formatting!

**Comment by felikatze** [comment]
@memeseller The new formatting got implemented this very week, so I don't blame you for missing it. The most up to date pages using it are Miraquest in acts/act3-4 and Meadow in rooms/dormont.

Since it adds a class to basically everything, I'd recommend using a shortcut program that types them out for you. Glow recommended Expanso to me, which I've had a good experience with doing the Miraquest page. I can also give you the file with the added shortcuts, if you want. Basically you just type (example) :dl and it autofills to <p class="dialogue-line"> which makes the everything more bearable. You can also configure your own shortcuts pretty easily.

Otherwise, it looks great!! You've even got the honor of making the first full page by someone who isn't me, so thank you so much!!

If you need more help, feel free to dm me @/felikatze on discord :sparkles:

**Comment by felikatze** [comment]
oh yeah, another thing i totally forgot - i'd also recommend adding some additional scenes to this page. Mainly: the sus variants of Odile's snack time dialogue, as well as the Odile last room convo leading up to the event. :thumbsup: Sadly events always end up being an interwoven mess of all sorts of things you gotta do. 

Also, concerning the specific conditions.  I added the Sus event and related things to the spreadsheet (realizing i got some details wrong that i added to your page, whoops.) (might also only be useful if you use rpgmaker mv to check things?)
[https://github.com/felikatze/isat-script-project/blob/main/spreadsheets/sidequest%20variables.ods]

TLDR: Read the Time Craft book in the candle room, read the wish craft books in the locked storage room on floor 1 and the secret library. have 15+ weird points when entering the last room.

Additionally, the dialogue you have under "not quite" actually always triggers in ACT 4 if the conditions weren't met, overriding the regular last room odile dialogue where bonnie interrupts with pineapple slices.

Sorry if this is a lot.

**Comment by memeseller** [comment]
@felikatze ahhh gotcha, that's good to know that it always triggers. Honestly when I was playing I went out of my way to be weird every new loop so I never noticed, haha. 

And yeah, I can work on adding those other scenes in here. I also checked the spreadsheet, and those variable names make sense (more or less) outside of rpgm. Unfortunately, I'm just using Notepad++ to extract and format the dialogue from the json files, so there's a bunch of random numbers there instead of nice variable names >_>

**Comment by goldstargloww** [comment]
@felikatze wanna look over this?

**Comment by felikatze** [comment]
@goldstargloww @memeseller i assumed we wanted to hold off on uploading the page until the other room dialogues are added (i can also do that, if yall want) but should we merge the page in its current state instead?

**Review by felikatze — APPROVED** [review]
